proxy/api/src/http/project.rs

```diff
@@ -167,14 +167,19 @@ mod handler {
         Ok(reply::json(&project::get(&ctx.state, urn).await?))
     }
 
-    /// Kick off a network request for the [`project::Project`] of the given `id`.
+    // TODO(sos): Should there be a `request` module that can take `POST`/`GET` requests for any
+    // `RadUrn` (i.e. `User` | `Project`)?
+    ///
+    ///  Kick off a network request for the [`project::Project`] of the given `id`.
     pub async fn request(ctx: context::Context, urn: coco::Urn) -> Result<impl Reply, Rejection> {
         // TODO(finto): Check the request exists in the monorepo
         let mut waiting_room = ctx.waiting_room.write().await;
-        let _request = waiting_room.request(urn, Instant::now());
+        let request = waiting_room.request(urn, Instant::now());
 
-        // TODO(finto): Serialise request and respond with that.
-        Ok(reply::json(&true))
+        // TODO(finto/sos): Serialise request and respond with that
```

Comment:
We could have it so that it returns `Either` where `Left` means it's a request that already existed and `Right` means that the request is fresh